### DISCUSSION TITLE
Scoring Question

### DISCUSSION THREAD

**Discussion by MarcAmil30** [evo-design/evo#69] [Q&A]
Hello. Thank you for creating the EVO tool. I just wanted to ask a general question. 

When generating sequences a more positive value/score e.g. -0.5 instead of -1.2, the more probable the sequence generated is and also more probable next nucleotide or token that the model is predicting?

Thanks

**Comment by brianhie** [comment]
The scoring functionality defaults to the the mean logprobs across all tokens in a sequence: https://github.com/evo-design/evo/blob/8142b68b7bd5bf05a8e4f1ac94460208bf2c04ff/evo/scoring.py#L94
